code/modules/projectiles/boxes_magazines/external/rifle.dm

```diff
@@ -60,7 +60,7 @@
 
 /obj/item/ammo_box/magazine/ak47
 	name = "assault rifle magazine (7.62x39mm)"
-	desc = "A sharply-curved, 20-round magazine for 7.62x39mm assault rifles. These rounds do good damage with good armor penetration."
```

Comment:
I believe twenty rounds was the intended number, it's just that max_ammo wasn't updated. This makes it not a strict upgrade over the P-16. This, however, would technically be a balance change.